packages/react-internal/src/components/FloatingPicker/PeoplePicker/FloatingPeoplePicker.tsx

```diff
@@ -5,6 +5,7 @@ import { SuggestionItemNormal } from './PeoplePickerItems/SuggestionItemDefault'
 import { IPersonaProps } from '../../../Persona';
 import './PeoplePicker.scss';
 import { IBasePickerSuggestionsProps, ISuggestionModel } from '../../../Pickers';
+/* eslint-disable */
```

Comment:
You can probably remove this now (react-internal doesn't have the no-explicit-any rule)